pints/toy/_eight_schools.py

```diff
@@ -0,0 +1,153 @@
+#
+# Eight schools log-pdf.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+from . import ToyLogPDF
+
+
+class EightSchoolsLogPDF(ToyLogPDF):
+    r"""
+    The classic Eight Schools example from [1]_. This model was used to
+    determine the effects of coaching on SATS scores in 8 schools. This model
+    is hierarchical and takes the form,
+
+    .. math::
+        \mu\sim\mathcal{N}(0, 5)
+        \tau\sim\text{Cauchy}(0, 5)
+        \theta_j\sim\mathcal{N}(\mu, \tau)
+        y_j\sim\mathcal{N}(\theta_j, \sigma_j),
+
+    where :math:`sigma_j` is known. The user may choose between the "centered"
+    parameterisation of the model (which exactly mirrors the statistical
+    model), and the "non-centered" parameterisation, which introduces
+    auxillary variables to improve chain mixing. The non-centered model takes
+    the form,
+
+    .. math::
+        \mu\sim\mathcal{N}(0, 5)
+        \tau\sim\text{Cauchy}(0, 5)
+        \tilde{\theta}_j\sim\mathcal{N}(0, 1)
+        \theta_j = mu + \tilde{\theta}_j \tau
+        y_j\sim\mathcal{N}(\theta_j, \sigma_j).
+
+    Note that, in the non-centered case, the parameter samples correspond to
+    theta_tilde rather than theta.
+
+    Extends :class:`pints.toy.ToyLogPDF`.
+
+    Parameters
+    ----------
+    mu : float
```

Comment:
These aren't right

Reply:
Not sure I understand, sorry? Can you elaborate?

Reply:
They're not the constructor arguments